### ISSUE TITLE
Auto region switch

### ISSUE THREAD

**Issue by thehughhefner** [MiSTer-devel/SMS_MiSTer#110]
Hi,
Would it be possible to implement an Auto region switch instead of manually switching the region on the OSD? Apparently, the rom header has a region section (see following link): https://www.smspower.org/Development/ROMHeader#RegionCode0x7fff05Bytes

**Comment by sorgelig** [comment]
i don't see any mention of PAL region

**Comment by birdybro** [comment]
Correct, I had previously assumed there was a PAL region, but it looks as though there is basically just Japan or Not-Japan in that list. Sega and some other Japanese companies in the 80's and early 90's often considered the regions to be "Domestic/Japan" and "Everywhere Else", and then they would have NTSC or PAL be decided by the cartridge and the hardware for that region combined.
